src/main/resources/templates/pps/bankTransferSanctionsDetails.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+
+<html xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layouts/baseLayout}">
+
+<head>
+    <title>
+        Pay by bank transfer - Confirmation statement
+    </title>
+</head>
+
+<div id="bank-transfer-confirmation-statement-main-content" layout:fragment="content">
+    <form th:action="@{''}" method="post" class="form" id="bankTransferConfirmStatement">
```

Comment:
id here would be similar e.g. `bank-transfer-sanctions-details-form`, so suggest a convention to have the id's ending in `-form` here and the enclosing div ending with `-main-content` in our new pages?